ui/src/pages/Converter/File/FileConverter.module.scss

```diff
@@ -2,4 +2,30 @@
 	display: flex;
 	flex-direction: column;
 	gap: 20px;
+
+	&__item {
+		min-height: 66px;
+		padding: 8px;
+		border: 1px solid #d9d9d9;
+		border-radius: 8px;
```

Comment:
I think this file is behind the main branch, please pull the latest changes, thanks!